app/controllers/socks_controller.rb

```diff
@@ -0,0 +1,53 @@
+class SocksController < ApplicationController
+
+    def index
+        @socks = Sock.all
```

Comment:
Please improve indentation in the entire file - 2 spaces for 1 level of indentation.